### PR TITLE
Fix APIM migration command and remove pre-validation step

### PR DESCRIPTION
**Pull request by msm1992**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Review by binodmx — APPROVED** [review]

File: en/docs/install-and-setup/upgrading-wso2-api-manager/320-to-410/upgrading-from-320-to-410.md [hunk 1]
```diff
@@ -344,29 +344,17 @@ You have to run the following migration client to update the API Manager artifac
...
-    ```
-
-    Check the server logs and verify are there any errors logs. If you have encounter any errors in  api definitions, you have to correct them manually on old version before the component migration. 
-
-5.  Start the API-M server to migrate the api-m components as follows.
+4.  Start the API-M server to migrate the api-m components as follows.
 
     ``` tab="Linux / Mac OS"
-    sh api-manager.sh -DmigrateFromVersion=3.2.0
+    sh api-manager.sh -Dmigrate  -DmigrateFromVersion=3.2.0 -DmigratedVersion=4.1.0
```
Comment (binodmx) [inline en/docs/install-and-setup/upgrading-wso2-api-manager/320-to-410/upgrading-from-320-to-410.md:350]: 2 spaces between -Dmigrate and -DmigrateFromVersion

File: en/docs/install-and-setup/upgrading-wso2-api-manager/320-to-410/upgrading-from-320-to-410.md [hunk 2]
```diff
@@ -344,29 +344,17 @@ You have to run the following migration client to update the API Manager artifac
...
     ```
     
     ``` tab="Windows"
-    api-manager.bat -DmigrateFromVersion=3.2.0
+    api-manager.bat -Dmigrate  -DmigrateFromVersion=3.2.0 -DmigratedVersion=4.1.0
```
Comment (binodmx) [inline en/docs/install-and-setup/upgrading-wso2-api-manager/320-to-410/upgrading-from-320-to-410.md:354]: 2 spaces between -Dmigrate and -DmigrateFromVersion

Comment (msm1992) [reply en/docs/install-and-setup/upgrading-wso2-api-manager/320-to-410/upgrading-from-320-to-410.md]: fixed with https://github.com/wso2/docs-apim/pull/5983/commits/a2f75cbe24436a71cac4728778c614c43d6614c0

File: en/docs/install-and-setup/upgrading-wso2-api-manager/320-to-410/upgrading-from-320-to-410.md (hunk 1 above)
Comment (msm1992) [reply en/docs/install-and-setup/upgrading-wso2-api-manager/320-to-410/upgrading-from-320-to-410.md]: fixed with https://github.com/wso2/docs-apim/pull/5983/commits/a2f75cbe24436a71cac4728778c614c43d6614c0

**Pull request merged**